README.md

```diff
@@ -127,9 +127,8 @@ postpone).
   - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
   a *summary comment* trying to lay out the current state of the discussion and
   major trade-offs/points of disagreement.
-  - Before actually entering FCP, *all* members of the subteam must sign off;
-  this is often the point at which many subteam members first review the RFC in
-  full depth.
+  - Before actually entering FCP, *all* maintainers must sign off; this is
```

Comment:
I think we should reconsider the "all maintainers" policy and move to a 2/3rds rule, or a similar majority rule. Otherwise we run the risk of being blocked by maintainers who are not active.

Reply:
See #6.

Reply:
I agree, requiring sign off from all the maintainers is not practical. This is actually why I wasn't sure the reference to subteam was merely an oversight. I thought it might refer to a subset of the maintainers.

Reply:
Why not to identify components and components owners so they will provide consent by signing-off?

Reply:
Many rfcs will span components. I'd prefer we go with a simple majority of maintainers here.

Reply:
Agree that we need a majority of maintainers agreeing. All is impractical and even 2/3s seems extreme.

Reply:
This PR is not meant to address this issue. This PR removes the subteam reference.

I will push a PR that closes #6 with a "majority" comment.